### DISCUSSION TITLE
Welches Koordinatensystem benutzen wir eigentlich

### DISCUSSION THREAD

**Discussion by victoriafoxgirl** [kwautztretschke/Zylinderino#22] [General]
https://github.com/kwautztretschke/Zylinderino/blob/956a1497315943644ca9330c8eb4b7adc210aeaa/src/zylHW.cpp#L36

Bisher haben wir x nach oben und y nach rechts, wie OpenGL so tut, aber manchmal benutzt man auch x nach unten und y nach rechts.

Is mir eigentlich Wumpe

**Comment by nandbert** [comment]
Üblich sind x nach rechts und y nach oben (OpenGL) oder y nach unten (Kamera, Tiefe ist positives Z in einem Rechtssystem) :grimacing:
